app/assets/javascripts/controllers/container_dashboard/card/heatmaps/heatmaps-card-directive.js

```diff
@@ -0,0 +1,23 @@
+angular.module('miq.card').directive('heatmapsCard', ['ChartsDataMixin' ,function(chartsDataMixin) {
+    'use strict';
+    return {
+        restrict: 'A',
+        scope: {
+            title: '@',
+            heatmaps: '=',
+            hidetopborder: '@',
+            heatmapChartHeight: '=',
+            columnSizingClass: '@',
+            heatMapUsageLegendLabels: '='
+        },
+        templateUrl: '/static/heatmaps-card.html',
+        controller: ['$scope',
+            function($scope, $rootScope) {
+                if ($scope.columnSizingClass === undefined) {
+                    $scope.columnSizingClass = "col-xs-8 col-sm-6 col-md-6";
+                }
+                $scope.noLabels = [];
+            }
+        ]
+    };
+}]);
```

Comment:
Correct @zeari , the heatmap and the heatmap legend are part of A-PF, but the card is not since it's easy enough to put charts in cards. However, there is some debate on making specific cards for different charts in future A-PF releases.